src/views/perms/const.js

```diff
@@ -113,6 +115,7 @@ export const AssetPermissionTableMeta = {
     }
   },
   accounts: {
+    width: 80,
     formatter: AmountFormatter,
     formatterArgs: {
       cellValueToRemove: ['@SPEC'],
```

Comment:
The code seems to be correct and there aren't notable discrepancies between the provided snippet for September 2021 and its updated version by May 2025. There appear to have been no updates made in subsequent years, but I'll provide this information anyway:

There isn't any apparent issue with the current structure of the code nor does it seem like there would be significant optimizations to make at this point based on what you've given.

In summary: the code is not outdated and appears to contain useful components that should run smoothly today without major modifications. No optimization recommendations or fixes needed.